zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -71,3 +76,25 @@
     .skip_configuration()
     .add_to_registry()
 )
+
+(
+    TuyaQuirkBuilder("_TZE200_pay2byax", "TS0601")  # Cusam ZG-102ZL
+    .applies_to("_TZE200_n8dljorx", "TS0601")
+    .tuya_sensor(
+        dp_id=101,
+        attribute_name="measured_value",
+        type=t.uint16_t,
+        fallback_name="Illuminance",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    )
+    .tuya_binary_sensor(
+        dp_id=1,
+        attribute_name="zone_status",
+        device_class=BinarySensorDeviceClass.OPENING,
+        fallback_name="Opening",
+        entity_type=EntityType.STANDARD,
```

Comment:
Also need `translation_key` here

Reply:
Not required here due to the same reason as [here](https://github.com/zigpy/zha-device-handlers/pull/3512/files#r1842444759). Zigpy also validates the `translation_key`, but as there's a `device_class` provided here, HA will use that.